README.rst

```diff
@@ -146,7 +146,7 @@ We first build and install FRUT with CMake: ::
   -- Generating done
   -- Build files have been written to: <root>/FRUT/build
 
-  $ cmake --build . --target install
+  $ cmake --build . --target install --parallel
```

Comment:
The `--parallel` option was added to CMake 3.12, so this might fail with `Unknown argument --parallel` when using an older CMake version. FRUT requires CMake 3.4 minimum, so it is quite likely that some users don't have the latest version.

Reply:
Makes sense. Thanks!